src/subscribables/dependentObservable.js

```diff
@@ -52,6 +51,29 @@ ko.computed = ko.dependentObservable = function (evaluatorFunctionOrOptions, eva
         }
     }
 
+    // evaluatePossiblyAsync is defined in the constructor so that it can access the computed properties
+    // and method using a closure instead of "this", meaning that we can pass the function to "subscribe"
+    // without the need to use "bind". Older versions of Firefox seem to use up a lot more stack space
+    // when using a bound function or using "call". See #1622.
+    var evaluationTimeoutInstance = null;
+    computedObservable.evaluatePossiblyAsync = function () {
```

Comment:
I'm okay with that, and as you say, going forward the use of deferred updates makes this issue go away.  @SteveSanderson ?

I think then the answer might be to just reduce the number of iterations to 100.  Legacy FF should pass with that (I think it succeeds with 130 or so, but not positive– if we go this route, I'll fix it up).